app/controllers/game_day_controller.rb

```diff
@@ -4,6 +4,7 @@ class GameDayController < ApplicationController
   def index
     @episode = Episode.where("air_date >= '#{Date.today}'").
                        sort { |a, b| a.number <=> b.number }.first
+    @players = @episode.season.picks.collect { |x| x.user }
```

Comment:
Prefer `map` over `collect`.<br>Pass `&:user` as an argument to `collect` instead of a block.